pkg/util/mount/exec_mount.go

```diff
@@ -144,14 +144,6 @@ func (m *execMounter) EvalHostSymlinks(pathname string) (string, error) {
 	return m.wrappedMounter.EvalHostSymlinks(pathname)
 }
 
-func (m *execMounter) PrepareSafeSubpath(subPath Subpath) (newHostPath string, cleanupAction func(), err error) {
-	return m.wrappedMounter.PrepareSafeSubpath(subPath)
-}
-
-func (m *execMounter) CleanSubPaths(podDir string, volumeName string) error {
-	return m.wrappedMounter.CleanSubPaths(podDir, volumeName)
-}
-
 func (m *execMounter) SafeMakeDir(pathname string, base string, perm os.FileMode) error {
```

Comment:
@jsafrane we added SafeMakeDir as part of the subpath work.  Should that go inside the subpath package even though it's more of a file util?

Reply:
+1. And take `DoSafeOpen`, `LockAndCheckSubPathWithoutSymlink` and similar functions with it, if possible. They're not related to mounter (neither subpaths, but that's the only reason they exist).

Reply:
👍